### PR TITLE
Fix/law66

### PR DESCRIPTION
**Pull request by whaeck**
Fixing an omission in the ACE format concerning ACE LAW = 66

### CODE REVIEW

File: src/ContinuousEnergyNeutron.tex [hunk 1]
```diff
@@ -961,6 +975,8 @@ \subsubsubsection{\var{LAW}=66---$N$-body phase space distribution}\label{sec:LA
   \label{eq:LAW66}
 \end{align}
 
+While MCNP will only use the values for \var{NPSX} and $A_{P}$ (given in \var{LDAT}(1) and \var{LDAT}(2)), NJOY does add the derived probability density function and cumulative density function for $T(\xi)$ starting at \var{LDAT}(3).
```
Comment (jlconlin) [inline src/ContinuousEnergyNeutron.tex:978]: 👍🏻

**Review by jlconlin — APPROVED** [review]

File: src/ContinuousEnergyNeutron.tex (hunk 1 above)
Comment (paulromano) [reply src/ContinuousEnergyNeutron.tex:978]: Has this always been true?!

Comment (whaeck) [reply src/ContinuousEnergyNeutron.tex:978]: As far as I know. Even the MCNP5 manual ACE format appendix only gives NPSX and AP as the ACE LAW = 66 XSS array and does not mention the xi part of the law. The NJOY source code does however write all these values to the ACe file.

Comment (paulromano) [reply src/ContinuousEnergyNeutron.tex:978]: Makes me wonder what other secrets are hidden in the ACE format :wink:

Comment (jlconlin) [reply src/ContinuousEnergyNeutron.tex:978]: Yeah, me too.

Comment (whaeck) [reply src/ContinuousEnergyNeutron.tex:978]: At least the incident neutron and incident charged particle ones don't seem to have any obscure specs left (I still need to write the specs for ACE LAW = 33 but we know it exists).

**Pull request merged**